mozilla/activity-stream
# Fixes bug 1507973 - v2 with attachments for personality provider in activity stream.

To test:
1. Make a new profile, but don't log into it.
2. Go into that profile's dir, and paste in a places.sqlite from an existing profile to get a good chunk of history to work with.
3. Create a user.js file with this in it:
```
user_pref("services.settings.server", "https://kinto.dev.mozaws.net/v1");
user_pref("services.settings.verify_signature", false);
```
4. Now run the profile.
5. In the browser debugger, paste in `Services.obs.notifyObservers(null, "idle-daily");`
6. Make an insignificant change to the pref browser.newtabpage.activity-stream.feeds.section.topstories.options to trigger a fresh cache.

Expected: you should now be seeing personalized stories on about:home.
